README.md

```diff
@@ -207,6 +207,29 @@ my_tour:
 * `action` is one of the Shepherd tour method names, i.e. `cancel`, `next`, or `complete`
 * `classes` are the CSS class names you want applied to the button
 
+### Flipper integration
+
+If you have Flipper installed as a dependency in your project you will be able to enable or disable tours based on a flipper using the `flipper_key` option. This will automatically enable a tour when this flipper is active and disable it when it's inactive.
```

Comment:
```suggestion
If you have [Flipper](https://github.com/jnunemaker/flipper) installed as a dependency in your project you will be able to enable or disable tours based on a flipper using the `flipper_key` option. This will automatically enable a tour when this flipper is active and disable it when it's inactive.
```